### ISSUE TITLE
Newest release v0.18.2 fails to find proper baseline in GH Action workflow

### ISSUE THREAD

**Issue by mgr0dzicki** [obi1kenobi/cargo-semver-checks#382]
### Steps to reproduce the bug with the above code

```
git clone https://github.com/mgr0dzicki/cargo-semver-action-ref-slice.git
cd cargo-semver-action-ref-slice
git checkout major_change
```

### Actual Behaviour

```
cargo install cargo-semver-checks --vers 0.18.1
cargo semver-checks check-release
```
results in
```
    Updating index
     Parsing ref_slice v1.2.2 (current)
     Parsing ref_slice v1.2.1 (baseline)
    Checking ref_slice v1.2.1 -> v1.2.2 (patch change)
   Completed [   0.020s] 41 checks; 40 passed, 1 failed, 0 unnecessary
```
but
```
cargo install cargo-semver-checks
cargo semver-checks check-release
```
outputs
```
     Parsing ref_slice v1.2.2 (current)
     Parsing ref_slice v1.2.2 (current)
     Parsing ref_slice v1.2.2 (baseline)
    Checking ref_slice v1.2.2 -> v1.2.2 (no change)
   Completed [   0.021s] 41 checks; 41 passed, 0 unnecessary
```

### Expected Behaviour

The selected baseline should be still ref_slice@1.2.1, as it is the latest one published on crates.io.

#### Software version

cargo-semver-checks 0.18.2

#### Operating system

Linux 5.10.154-1-MANJARO

#### Command-line

```bash
/home/mieszko/.cargo/bin/cargo-semver-checks semver-checks --bugreport 
```

#### cargo version

```
> cargo -V 
cargo 1.66.0-beta.2 (d65d197ad 2022-11-15)
```

#### Compile time information

- Profile: release
- Target triple: x86_64-unknown-linux-gnu
- Family: unix
- OS: linux
- Architecture: x86_64
- Pointer width: 64
- Endian: little
- CPU features: fxsr,sse,sse2
- Host: x86_64-unknown-linux-gnu

**Comment by obi1kenobi** [comment]
I think I also ran into this yesterday, and I think #381 is going to fix it. I'll add this check to CI as well.

The issue is that the library PR accidentally made the default baseline behavior "use the current directory as the root" instead of making it search for a version on crates.io.

**Comment by mgr0dzicki** [comment]
It indeed fixed the issue: https://github.com/obi1kenobi/cargo-semver-checks-action/actions/runs/4242339730, thank you!

**Comment by obi1kenobi** [comment]
v0.18.3 is published with the fix, and I've yanked all prior 0.18.x releases.

Thanks for the report!
